samples/local_attestation/common/attestation.cpp

```diff
@@ -21,36 +28,35 @@ bool Attestation::generate_local_report(
     const uint8_t* data,
```

Comment:
Change the function name from generate_local_report to generate_local_attestation_evidence